src/schema/v2/sale/index.ts

```diff
@@ -137,7 +143,13 @@ export const SaleType = new GraphQLObjectType<any, ResolverContext>({
             gravityArgs.exclude_ids = flatten([options.exclude])
           }
 
-          const { body, headers } = await saleArtworksLoader(id, gravityArgs)
+          let loader
+          if (accessToken) {
```

Comment:
I wanted to keep branching like this b/c `saleArtworksLoader` is different whether `accessToken` is present or not.

An alternative might be `const loader = cached || accessToken ? saleArtworksLoader: uncachedSaleArtworksLoader`